planet/src/main/java/com/google/planet/servlets/EventServlet.java

```diff
@@ -74,6 +75,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         int openingTime = TimeRange.getTimeInMinutes(8,0);
         int closingTime = TimeRange.getTimeInMinutes(17,0);
 
+        // Get the count of the current list (will need to add more filters, such as userid, listName, etc. later)
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        PreparedQuery entityStats = datastore.prepare(new Query("Event"));
+        long numOfEvents = 0;
+        for (Entity entity : entityStats.asIterable()) {
+            numOfEvents += 1;
+        }
```

Comment:
I think there is a way to retrieve the number of entities without having to loop through all the elements. Check this: https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/PreparedQuery.html#countEntities-com.google.appengine.api.datastore.FetchOptions-

Reply:
Will try! I saw another method but it can only be used in production mode. Hopefully this one works

Reply:
It worked!